devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -1418,6 +1601,14 @@ void WholeBodyDynamicsDevice::readContactPoints()
     int numberOfContacts=0;
     size_t nrOfSubModels = estimator.submodels().getNrOfSubModels();
 
+    //construct the wrenches from the global variables
+    wholeBodyDynamics::UnknownWrenchContact wrench[overrideContactFrames.size()];
```

Comment:
why do we have to create local variables in multiple functions? Any specific rationale?

Reply:
removed and global variables are created instead in https://github.com/robotology/whole-body-estimators/pull/48/commits/edcd509dad868124ec98e627ecd260af425c8ca1